elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -78,7 +78,7 @@ class PfeAccordion extends PFElement {
   }
 
   // Each set contains a header and a panel
-  static get template() {
+  static get contentTemplate() {
```

Comment:
`this.template` exists in PFElement so this naming could conflict with that variable.

Reply:
good find!